src/workers/notifications/postAdded.ts

```diff
@@ -79,11 +79,13 @@ const worker: NotificationWorker = {
           select: { mentionedUserId: true },
           where: { postId: post.id },
         });
-        const members = await getSubscribedMembers(
+
+        const members = await getSubscribedMembers({
           con,
-          NotificationType.SquadPostAdded,
-          source.id,
-          {
+          type: NotificationType.SquadPostAdded,
+          byStatus: NotificationPreferenceStatus.Subscribed,
```

Comment:
Shouldn't this also be byNotMuted?

Reply:
No, here we want to select all members with status subscribed, i.e. opt-in.

`byNotStatus` is selecting all users who _do not_ have the status

Reply:
Ah ok, but don't you want subscribed OR notify status?

Reply:
No, just need notification preference status. It selects all members from source_member, then filters out where preference status is subscribed.

Ended up not using content_preference table, since we'd need to switch everything over so we don't accidentally miss somewhere 🙈

Reply:
Also uses "same" logic as notifications from machine sources

Reply:
Ah ok, I thought we had 2 status, subscribed or notified?

Reply:
No, `NotificationPreferenceStatus` is just `muted` or `subscribed`

```ts
export enum NotificationPreferenceStatus {
  Muted = 'muted',
  Subscribed = 'subscribed',
}
```